README.md

```diff
@@ -102,9 +102,20 @@ gh repo list OpenNeuroDatasets-JSONLD --fork -L 500 | awk '{print $1}' | sed 's/
 ```
 
 ## Running the `bagel-cli` on bulk annotated data
-The following scripts are needed:
+The following scripts are used:
 - `extract_bids_dataset_name.py`
-- `parallel_bagel.sh`
+- `add_description.py`
 - `run_bagel_cli.sh`
+- `parallel_bagel.sh`
+
+### Steps
+1. Activate a new Python environment and install the dependencies for this repo with `pip install -r requirements.txt`.
```

Comment:
```suggestion
1. (Optional) create a new Python environment with `python -m venv my_env`.
2. Activate your python environment with `source ./my_env/bin/activate`
3. Install the dependencies with `pip install -r requirements.txt`
```

Reply:
I think if we mention venvs and installations, we should go all the way with the explanation